packages/client/src/base/editor-context-service.ts

```diff
@@ -143,6 +143,14 @@ export class EditorContextService implements IActionHandler, Disposable, IDiagra
         return this.selectionService.onSelectionChanged;
     }
 
+    /**
+     * Event that is fired when the viewport of the diagram changes i.e. after the `CommandStack` has processed a viewport update.
+     * By default, this event is only fired if the viewport was changed via a `SetViewportCommand` or `BoundsAwareViewportCommand`
+     */
+    get onViewportChanged(): Event<Readonly<Viewport>> {
```

Comment:
I wonder whether we should also introduce a listener type like we have for:
- modelRootChanged -> TYPES.IGModelRootListener
- editModeChanged -> TYPES.IEditModeListener

What do you think?